api/client/nic.cc

```diff
@@ -358,19 +358,23 @@ NicAddOptions::NicAddOptions() {
     bypass_filtering = "false";
 }
 
+static inline bool IsArg(int count, int argc, char **argv, char const *option) {
+    return (count + 1 < argc && string(argv[count]) == option);
```

Comment:
I would write this
`return (count + 1) < argc && !strcmp(argv[count]), option);`
not everyone know C++ Operator Precedence, but the parentheses around return value are clary not useful.
http://en.cppreference.com/w/cpp/language/operator_precedence